matroska_tags.xml

```diff
@@ -252,25 +252,24 @@ an age in other countries or a URI defining a logo).</description>
       <description lang="en">Information on when the file was purchased; see also (#commercial) on purchase tags.</description>
     </tag>
     <tag name="RECORDING_LOCATION" class="Spatial Information" type="UTF-8">
-      <description lang="en">The location where the item was recorded. The countries corresponding to the string,
-same 2 octets country-codes as in Internet domains [@!IANADomains] based on [@!ISO3166-1] alpha-2 codes.
+      <description lang="en">The location where the item was recorded,
+in the Matroska countries form; see (#country-codes) on countries codes.
```

Comment:
"countries codes" should be "country codes", just like "language codes": you only pluralize the second word.